src/pyrtcm/rtcmhelpers.py

```diff
@@ -98,7 +96,7 @@ def bits2val(att: str, scale: float, bitfield: int) -> object:
     # apply any scaling factor
     else:
         if scale not in (0, 1):
-            val = round(val * scale, SCALEDP)
+            val *= scale

```

Comment:
Would it not be possible to round to a specified number of significant digits, rather than a specified number of decimal places?

Reply:
Possible e.g. ...

```
round_to_n = lambda x, n: x if x == 0 else round(x, -int(math.floor(math.log10(abs(x)))) + (n - 1))
```

... but computationaly slightly expensive and doesn't seem worthwhile in this instance. But, yes, any more than 8 significant digits is probably spurious.